### ISSUE TITLE
Bitwarden client installed with snapd not showing tray icon in Fedora Workstation 37

### ISSUE THREAD

**Issue by lucasricci** [bitwarden/clients#4195]
### Steps To Reproduce

1. Install Bitwarden using `snap install bitwarden`
2. Login into your account
3. Go to the settings
4. Enable tray icon and close to tray icon
5. Close the application, and the tray icon do not appear to work

### Expected Result

The expected result is to the Bitwarden client to minimize to the tray icon

### Actual Result

It simply doesn't work

### Screenshots or Videos

https://user-images.githubusercontent.com/62891523/206248640-206ee0a1-11c1-4ba1-b33d-6c9a1b3d6be0.mp4

### Operating System

Linux

### Operating System Version

Fedora Workstation 37 x86_64

### Installation method

Snap

### Build Version

2022.11.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by monkeyhybrid** [comment]
I'm not sure if I'm seeing the same issue on Kubuntu 22.04, but it may be related.

Bitwarden client has been working fine for me up until the latest update (2023.4.0) that I installed on the 27th April. Since that update, the Bitwarden client does reserve a place in my system tray but 'appears' as a blank space (ie. invisible icon). The only way to know it is actually there is by clicking the blank space in my system tray until I see I can see the Bitwarden context menu appear.

TL;DR - For me, the system tray functionality works but the Bitwarden icon is invisible.
